### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by niv-eini-hs**
### Summary
HSBot - Repository cleanup

### Jira ticket
https://hiredscore.atlassian.net/browse/DEV-17883

### CODE REVIEW

**Review by shaharb-hs — APPROVED** [review]
נראה לי פספת שם איזה נקודה :)

File: config/jest.shared.config.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-module.exports = {
```
Comment (ori-blau-hs) [inline config/jest.shared.config.js:1]: @niv-eini-hs same here, I don't think we need we need to touch configs, they usually auto generated by the nest framework

File: src/modules/hsbot/hs-demo-bots.module.ts [hunk 2]
```diff
@@ -14,13 +14,7 @@ import {HsDemoBotActivityHandlerService} from "./services/hsdemobot/hs-demo-bot-
 import {HsDemoBotsActivityHandlerModule} from "./hs-demo-bots-activity-handler.module";
```
Comment (ori-blau-hs) [inline src/modules/hsbot/hs-demo-bots.module.ts:14]: @niv-eini-hs Same comment here regarding the necessarity of demo files

**Review by yuval-chen-hs — APPROVED** [review]
yay

File: src/modules/hsbot/usecases/internalMobilityDemo/internalMobilityCards/welcome-internal-bot-demo-closure.ts [hunk 3]
```diff
@@ -37,9 +37,7 @@ export function welcomeInternalBotDemoClosure(
                 "type": "Action.OpenUrl",
```
Comment (ori-blau-hs) [inline src/modules/hsbot/usecases/internalMobilityDemo/internalMobilityCards/welcome-internal-bot-demo-closure.ts:37]: @niv-eini-hs Do we need this file?

Comment (niv-eini-hs) [reply src/modules/hsbot/usecases/internalMobilityDemo/internalMobilityCards/welcome-internal-bot-demo-closure.ts:37]: @boris-mikhailovski-hs is this file needed? did we miss it?

File: src/modules/hsbot/controllers/hs-demo-bot.controller.ts [hunk 4]
```diff
@@ -21,7 +21,6 @@ import {AllExceptionsFilter} from "../../../exception-filters/all.exception-filt
 import {logger} from "../../../libs/logger/loggerHandler";
```
[the remark this answers is not in the document]
Comment (niv-eini-hs) [reply src/modules/hsbot/controllers/hs-demo-bot.controller.ts:21]: This is the main module we need, we cannot remove it

File: src/modules/hsbot/hs-demo-bots.module.ts (hunk 2 above)
Comment (niv-eini-hs) [reply src/modules/hsbot/hs-demo-bots.module.ts:14]: This is the main module we need, we cannot remove it

File: config/jest.local.config.js [hunk 5]
```diff
@@ -1,9 +0,0 @@
-module.exports = {
```
[the remark this answers is not in the document]
Comment (niv-eini-hs) [reply config/jest.local.config.js:1]: Let's remove it and if we decide to use it let's return it. I wanted to strip it down before we do anything so we know the reason for everything and have less redundant files.

**Review by gabrielle-simon — APPROVED** [review]

**Pull request merged**